app/scripts/proactive/model/Task.js

```diff
@@ -381,11 +397,110 @@ define(
                 ]
 
                 this.controlFlow = {};
-                this.on("change", function (eventName, event) {
-                    undoManager.save()
+
+                // get the GI of the Task
+                var genericInformation = this.get["Generic Info"];

```

Comment:
I think it would be better to create a variable that will contain the value "Generic Info" instead of rewriting it each time.